pkg/plugin/rpc_test.go

```diff
@@ -159,7 +159,7 @@ func TestAccessRequesterRPC(t *testing.T) {
 		defer f.cancel()
 		expectedErr := "grant access error"
 		f.accessRequesterMock.EXPECT().GrantAccess(mock.Anything, mock.Anything).
-			Return(nil, fmt.Errorf(expectedErr))
+			Return(nil, fmt.Errorf("%s", expectedErr))
```

Comment:
`expectedErr.Error()`?

Reply:
`errors.New(expectedErr)`. The linter is happy with that